src/current/v24.2/changefeed-sinks.md

```diff
@@ -201,6 +201,35 @@ See the [Changefeed Examples]({% link {{ page.version.version }}/changefeed-exam
 
 {% include {{ page.version.version }}/cdc/note-changefeed-message-page.md %}
 
+## Amazon MSK Serverless
+
+{{site.data.alerts.callout_info}}
+CockroachDB instances must be in the same VPC as the MSK Serverless cluster in order for the changefeed to authenticate successfully.
+
+If you are running a CockroachDB {{ site.data.products.dedicated }} cluster, you must request egress private endpoints from your Cockroach Labs account team.
```

Comment:
Is this all correct? This is my understanding of the status quo as of a few weeks ago. Please let me know if I need to change this.

Reply:
my understanding after reading the wiki: https://cockroachlabs.atlassian.net/wiki/spaces/PM/pages/2746777681/Egress+Private+Endpoints+Manual+Preview is that egress private endpoints are only relevant to CC customers